.github/workflows/run-native-install.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 9 * * *"
+
+jobs:
+  native_install:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        release: ["lilac"]
+    env:
+      OPENEDX_RELEASE: open-release/${{ matrix.release }}.master
+
+    steps:
+      - name: Remove MySQL
+        run: |
+          sudo service mysql stop
+          sudo apt-get remove --purge mysql-server mysql-client mysql-common -y
+          sudo apt-get autoremove
+          sudo apt-get autoclean
+          sudo rm -rf /var/lib/mysql
+      - name: Remove MongoDB
+        run: |
+          sudo service mongod stop
+          sudo apt-get purge mongodb-org*
+          sudo rm -r /var/log/mongodb
+          sudo rm -r /var/lib/mongodb
+      - name: Ansible Bootstrap
+        run: wget https://raw.githubusercontent.com/edx/configuration/$OPENEDX_RELEASE/util/install/ansible-bootstrap.sh -O - | sudo -E bash
+      - name: Generate passwords
+        run: |
+          wget https://raw.githubusercontent.com/edx/configuration/$OPENEDX_RELEASE/util/install/generate-passwords.sh -O - | bash
+          ls -l
+      - name: Create config.yml
+        run: |
+          echo -e "EDXAPP_LMS_BASE: '$(curl ipinfo.io/ip)'\nEDXAPP_CMS_BASE: '$(curl ipinfo.io/ip):18010'" > config.yml
+          ls -l
+      - name: Native Install
+        run: wget https://raw.githubusercontent.com/edx/configuration/$OPENEDX_RELEASE/util/install/native.sh -O - | bash
```

Comment:
Currently getting this error

`npm update check failed                  │\n│           Try running with sudo or get access            │\n│           to the local update config store via           │\n│ sudo chown -R $USER:$(id -gn $USER) /home/runner/.config │\n└──────────────────────────────────────────────────────────┘\n`


No Failure in the Action step though because of #59 .